### PR TITLE
Replace method of generating UUIDs

### PR DESCRIPTION
**Pull request by kr8n3r**
What
----

Replace the use of `uuid` package for generating and validating UUIDs with NodeJS `crypto.randomUUID` and custom test function to check UUID validity.

Why
----

We'd like to reduce the number of dependencies we use for maintenance reasons and take advantage of native methods (if available) or own code if feasible. 

Added bonus: `randomUUID` is marginally faster, but negligible for our use-case

Assumption 
----

- we only need to use [UUID version 4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) (which is what `crypto.randomUUID` generates)

How to review
-------------

- review by commit
- run locally or deploy to dev env

Who can review
---------------

not @kr8n3r 

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: src/layouts/helpers.tsx [hunk 1]
```diff
@@ -72,3 +72,11 @@ export function conditionallyDisplay(
 export function capitalize(value: string): string {
   return value.charAt(0).toUpperCase() + value.slice(1);
 }
+
+/* Check if string is a valid UUID */
+export function checkIfValidUuid(input: string):boolean {
+  // 8-4-4-4-12 arrangement of 36 characters
+  const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
```
Comment (AP-Hunt) [inline src/layouts/helpers.tsx:79]: If you cast the string to upper case, you could simplify the regular expression

**Review by paroxp — APPROVED** [review]
Yes.

Let's kill all the dependencies.

File: src/layouts/helpers.tsx (hunk 1 above)
Comment (paroxp) [reply src/layouts/helpers.tsx:79]: Nice touch Andy. Better for readability of "write-only" regex's.

Comment (kr8n3r) [reply src/layouts/helpers.tsx]: this in mind?

Comment (AP-Hunt) [reply src/layouts/helpers.tsx]: Precisely what I was thinking, yeah

**Pull request merged**